examples/cifar10_to_monai.ipynb

```diff
@@ -0,0 +1,606 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Adding MONAI to PyTorch cifar10 classification"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This notebook will show the flexibility of MONAI by taking a very simple PyTorch example and enhancing it with MONAI. The example we will begin with is classification on CIFAR10 dataset (https://pytorch.org/tutorials/beginner/blitz/cifar10_tutorial.html). This notebook is broken into following parts:\n",
+    "\n",
+    "1. The original CIFAR-10 classification tutorial.\n",
+    "2. Using MONAI transforms without having to change any code.\n",
+    "3. Easily replacing a NueralNetwork with one provided in MONAI."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Setup"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "CIFAR-10 is a classification dataset with 10 classes ‘airplane’, ‘automobile’, ‘bird’, ‘cat’, ‘deer’,\n",
+    "‘dog’, ‘frog’, ‘horse’, ‘ship’, ‘truck’. Each image in CIFAR-10 is 3x32x32."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "env: CUDA_VISIBLE_DEVICES=1\n"
```

Comment:
No need to set CUDA visible device in this tutorial, if user only has 1 GPU, it may cause an error to set =1.

Reply:
makes sense. i'll fix this.